src/Literate.jl

```diff
@@ -123,7 +123,7 @@ function parse(content; allow_continued = true)
 end
 
 function replace_default(content, sym;
-                         config::Dict,
+                         config::AbstractDict,
```

Comment:
This one is internal and after the arguments is processed to a `Dict`:
```suggestion
                         config::Dict,
```